graphql/version_resolver.go

```diff
@@ -69,7 +69,7 @@ func (r *versionResolver) BuildVariants(ctx context.Context, obj *restModel.APIV
 	}
 	groupedBuildVariants, err := generateBuildVariants(ctx, utility.FromStringPtr(obj.Id), options, utility.FromStringPtr(obj.Requester), r.sc.GetURL())
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error generating build variants for version %s : %s", *obj.Id, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("generating build variants for version %s : %s", *obj.Id, err.Error()))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("generating build variants for version '%s': %s", *obj.Id, err.Error()))
```